src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json

```diff
@@ -12,7 +12,7 @@
   "PostResponseScript": "$Response = $Response.target",
   "VersionRegex": "[0-9.]+",
   "InstallerRegex": ".(exe|msi|msix|appx)(bundle){0,1}$",
-  "PreviousVersion": "3.9.4.403",
```

Comment:
The API breaks on setting the version `3.10.10.402`.

Reply:
@SpecterShell if this works in your region, I can merge this PR.

Reply:
They rolled back to `3.10.10.402` just now, which causes the API returns nothing. At the time I was testing it worked fine.